executor/config/config.py

```
@@ -152,3 +152,30 @@ def get_wegent_mcp_url() -> str:
         The Streamable HTTP endpoint URL for the MCP server.
     """
     return f"{WEGENT_MCP_URL}"
+
+
+def sync_device_config(device_config: "DeviceConfig") -> None:
```

Ruff (0.15.2)

[error] Undefined name `DeviceConfig`

(F821)